packages/jsii-pacmak/test/expected.jsii-calc/dotnet/Amazon.JSII.Tests.CalculatorPackageId/Amazon/JSII/Tests/CalculatorNamespace/Submodule/Child/Structure.cs

```diff
@@ -0,0 +1,25 @@
+using Amazon.JSII.Runtime.Deputy;
+
+#pragma warning disable CS0672,CS0809,CS1591
+
+namespace Amazon.JSII.Tests.CalculatorNamespace.Submodule.Child
+{
+    #pragma warning disable CS8618
```

Comment:
Why is this needed? Curious...

Reply:
`CS8618` is a warning about non-nullable fields that have no initializer... This warning is immensely difficult to avoid in the generated code because of how we use dynamic properties... There's no initial assignment because the value is never stored in C# (it is read from and written to the JS process).

The disable is emitted systematically in all generated files so that stuff compiles "cleanly".

Reply:
ok. thanks